internal/pipeline/step_command_matrix_test.go

```diff
@@ -0,0 +1,367 @@
+package pipeline
+
+import (
+	"errors"
+	"testing"
+)
+
+func TestMatrix_ValidatePermutation_Simple(t *testing.T) {
+	t.Parallel()
+
+	matrix := &Matrix{
+		Setup: MatrixSetup{
+			"": {
+				"Apiaceae",
+				"Brassicaceae",
+				"Cucurbitaceae",
+				"Fabaceae",
+				"Lamiaceae",
+				"Rosaceae",
+				"Rutaceae",
+				"Solanaceae",
```

Comment:
you know how much i love taxonomy (a lot)... but could we either `s/ceae//g` or (gasp) use the common names for these families? the repeated suffixes makes it hard for me to read

Reply:
also, i love a new addition to the Buildkite Agent Test Data Cinematic Universe